src/xcode/ENA/ENA/Source/Developer Menu/Features/DMDebugRiskCalculationViewController.swift

```diff
@@ -82,32 +67,26 @@ class DMDebugRiskCalculationViewController: UIViewController {
 			view.bottomAnchor.constraint(equalTo: textView.bottomAnchor)
 		])
 
-		guard let mostRecentRiskCalculationConfiguration = store.mostRecentRiskCalculationConfiguration,
-			  let mostRecentRiskCalculation = store.mostRecentRiskCalculation else {
-			textView.text = "No risk calculation run yet."
+		guard let mostRecentRiskCalculationConfiguration = store.mostRecentRiskCalculationConfiguration else {
+			textView.text = "missing configuration - stop here"
```

Comment:
Why not leave the old guard here that checks for both values and otherwise shows "No risk calculation run yet."?

Reply:
Then `mostRecentRiskCalculation` in `RiskCalculationDebugHelper` wouldn't have to be optional, they are always both there or none of them is anyway.